lib/plan-builder-generated.js

```diff
@@ -1115,7 +1115,7 @@ point(...args) {
     const paramdefs = [['latitude', [types.XsDouble, PlanColumn, PlanParam], true, false], ['longitude', [types.XsDouble, PlanColumn, PlanParam], true, false]];
     const checkedArgs = (namer !== null) ?
         bldrbase.makeNamedArgs(namer, 'cts.point', 2, new Set(['latitude', 'longitude']), paramdefs, args) :
-        bldrbase.makePositionalArgs('cts.point', 2, false, paramdefs, args);
+        bldrbase.makePositionalArgs('cts.point', 1, false, paramdefs, args);
```

Comment:
Sorry but one last thing - any manual modifications you make to this file should have a comment on them explaining that they were manually made. That way, when we run the (hopefully not buggy) generator next time, we'll be able to see in the diff that our manual code is being removed, but the comment on it will tell us not to remove that code.